app/models/admin/courts/builder.js

```diff
@@ -33,7 +33,49 @@ const CourtRemovedViewModel = (court) => {
   })
 }
 
+const PoliceOffersAddedViewModel = (officers) => {
```

Comment:
This should probably be in a file for police (not courts) but not going to quibble as we don't have a lot of time this week :)